### DISCUSSION TITLE
Bosch Dishwasher - Energy and Water entities?

### DISCUSSION THREAD

**Discussion by helpfinder** [ekutner/home-connect-hass#449] [General]
Hi folks,
I used this HACS integration together with my Bosch Dishwasher machine (SPS4EMI24E). All good, appliance connected, but I do not understand some entities
e.g. I have Bosch Dishwasher - Energy reporting 37% or Bosch Dishwasher - Water reporting 49% > not sure how to interpret these values. Percentage makes no sense for me.

Thanks

**Comment by felix-albrecht** [comment]
see here: https://github.com/ekutner/home-connect-hass/discussions/451
